.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Ref: https://help.github.com/articles/about-codeowners/
```

Comment:
hmm are you requesting from this file that I add you as co-maintainer?

I wouldn't mind giving you write acess if you want to keep maintaining the code, since I'm not using or maintaining it myself.